tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -230,22 +229,27 @@ class SPIRVEmitter : public ASTConsumer {
     }
   }
 
-  void doStmt(Stmt *stmt) {
-    if (auto *retStmt = dyn_cast<ReturnStmt>(stmt)) {
+  void doStmt(const Stmt *stmt) {
```

Comment:
I'm leaning towards to get away with no comments on these `do*Stmt`/`do*Expr` methods. If we add comments, they are basically all "translate the given expression into SPIR-V (and return the result id)", which isn't particularly helpful. Comments should not be needed when the code are clear enough. But for logic, I do think we should put abundant comments to describe it. That's also what I have been doing. : D

How do you think?